lib/scope.js

```diff
@@ -94,6 +94,10 @@ SymbolDef.prototype = {
     }
 };
 
+AST_Toplevel.DEFMETHOD("is_block_scope", function(){
```

Comment:
This must be defined before `figure_out_scope` or else it's not defined. Not sure why.